Automattic/wp-calypso
# Plugin cards: Padding

#### Proposed Changes

Update plugin cards padding according to f84fqtCG4pCnCh2Hq0But5-fi-5722%3A41188&t=r33NcvXkkmLX8SW5-1

#### Testing Instructions
* Go to `/plugins`
* Check if the padding matches the ones described on Figma

#### Pre-merge Checklist





Fixes #72391
